python/graphstorm/gconstruct/file_io.py

```diff
@@ -102,6 +102,16 @@ def write_data_csv(data, data_file, delimiter=','):
     data_frame = pd.DataFrame(data)
     data_frame.to_csv(data_file, index=True, sep=delimiter)
 
+def _pad_stack(arrs):
+    max_len = max(len(arr) for arr in arrs)
+    dtype = arrs[0].dtype
+    for i, arr in enumerate(arrs):
```

Comment:
Will this implementation more efficient?
```
new_arrs = np.zeros((len(arr), max_len), dtype=dtype)
for i, arr in enumerate(arrs):
   new_arrs[i][[:len(arr)] = arr
```
So we do not need to allocate many small numpy arrays.